extensions/wikia/ArticleVideo/scripts/ooyalaVideo.js

```diff
@@ -0,0 +1,39 @@
+define('ooyalaVideo', ['jquery', 'wikia.window', 'wikia.loader'], function ($, window, loader) {
```

Comment:
It worries me a bit that we create yet another handler for Ooyala. For example, the AdEng team will have to rethink how we load ads there: https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/template/ooyala.js. This might be fine for the MVP but we should consider having a shared handler in the long run.